helm/vitess/templates/_orchestrator-conf.tpl

```diff
@@ -77,7 +78,7 @@ data:
     ],
     "PostMasterFailoverProcesses": [
         "echo 'Recovered from {failureType} on {failureCluster}. Failed: {failedHost}:{failedPort}; Promoted: {successorHost}:{successorPort}' >> /tmp/recovery.log",
-        "vtctlclient -server vtctld.{{ $namespace }}:15999 TabletExternallyReparented {successorAlias}"
+        "vtctlclient {{ include "format-flags-inline" $defaultVtctlclient.extraFlags | replace "\"" "\\\"" }} -server vtctld.{{ $namespace }}:15999 TabletExternallyReparented {successorAlias}"
```

Comment:
Would it work to use the `toJson` function to do this escaping?
http://masterminds.github.io/sprig/defaults.html

Reply:
Good catch, I changed it to use `toJson` in conjunction with `trimAll` to remove the outer `"`, which looks a lot cleaner. https://github.com/vitessio/vitess/pull/4333/commits/8f4446e5e81152911e2b18b8efc6af0b9b29a1bb